output/schema/schema.json

```diff
@@ -164972,8 +164972,11 @@
           }
         },
         {
-          "name": "human",
+          "description": "If true, returns the name of each index in each snapshot.",
+          "name": "index_names",
           "required": false,
+          "serverDefault": true,
+          "since": "7.13.0",
```

Comment:
I believe this was only introduced in 8.3.0?

https://github.com/elastic/elasticsearch/pull/86269

Reply:
Thanks for correcting this! I'll get the right version there.